roles/build-release/tasks/build.yaml

```
@@ -58,6 +58,7 @@
       {{ _feature_freeze | default('') }}
       {{ '--tags-file' if antsibull_tags_validate else '' }}
       {{ '--preserve-deps' if antsibull_preserve_deps else '' }}
+      {{ '--ignore-version-regressions' if antsibull_ignore_version_regressions else '' }}
```

I have not looked at the whole PR yet, but I'll note that we probably need to explicitly coerce these values to booleans like we did in the rest of the role in the last antsibull-build release.

```suggestion
      {{ '--ignore-version-regressions' if antsibull_ignore_version_regressions | bool else '' }}
```

---

I added some more `| bool`s as well, it looks like we didn't caught all of them last time.